template/manifests/PodDisruptionBudget/manifest/draft.yaml

```diff
@@ -0,0 +1,52 @@
+templateName: "podDisruptionBudget-manifest"
+description: "This template is used to create a PodDisruptionBudget for an application"
+type: "podDisruptionBudget"
+variables:
+  - name: "APPNAME"
+    type: "string"
+    kind: "resourceName"
+    description: "the name of the application"
+    default:
+      value: "frontend"
+  - name: "PARTOF"
+    type: "string"
+    kind: "part-of"
+    description: "the label to identify which project the resource belong to"
+    default:
+      value: "online-store"
+  - name: "GENERATORLABEL"
+    type: "string"
+    kind: "label"
+    description: "the label to identify who generated the resource"
+    default:
+      value: "draft"
+  - name: "MINIMUMREPLICAS"
+    type: "int"
+    kind: "replicas"
```

Comment:
I think `replicaCount` would be a better kind definition for this and maxreplicas